src/main/scala/com/kaizen/movie/Http.scala

```diff
@@ -0,0 +1,32 @@
+package com.kaizen.movie
+
+import caliban.{CalibanError, GraphQLInterpreter, Http4sAdapter}
+import cats.Monad
+import cats.data.Kleisli
+import cats.effect.{Blocker, ConcurrentEffect, ContextShift, Sync, Timer}
+import org.http4s.server.Router
+import org.http4s.server.blaze.BlazeServerBuilder
+import org.http4s.{HttpRoutes, StaticFile}
+import org.http4s.implicits._
+
+import scala.concurrent.ExecutionContext
+import scala.concurrent.ExecutionContext.global
+
+object Http {
+  def server[F[_]: ConcurrentEffect: ContextShift : Timer](blockingEC: ExecutionContext, gqlRoute: HttpRoutes[F]): F[Unit] = {
```

Comment:
I believe we can use the `Blocker` module here directly instead of exposing the underlying `ExecutionContext` as it is safer. WDYT?

Reply:
Also, I notice the use of Tagless Final, do we want to just commit to `zio.Task` or would you rather go this route?